sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_event_v1/query.sql

```diff
@@ -3,19 +3,25 @@ SELECT
   sample_id,
   client_id,
   COUNT(*) AS n_logged_event,
-  COUNTIF(
-    event_category = 'pictureinpicture'
-    AND event_method = 'create'
-  ) AS n_created_pictureinpicture,
-  COUNTIF(
-    event_category = 'security.ui.protections'
-    AND event_object = 'protection_report'
-  ) AS n_viewed_protection_report,
+  COUNTIF( event_category = 'pictureinpicture'
+    AND event_method = 'create' ) AS n_created_pictureinpicture,
+  COUNTIF( event_category = 'security.ui.protections'
+    AND event_object = 'protection_report' ) AS n_viewed_protection_report,
+  ARRAY_AGG(
+  IF
+    (event_category = 'contextservices.quicksuggest'
+      AND event_method = 'opt_in_dialog',
+      event_object,
+      NULL) IGNORE NULLS
+  ORDER BY
+    session_start_time DESC )[
+OFFSET
+  (0)] AS firefoxsuggest_optin,
 FROM
   telemetry.events
 WHERE
   submission_date = @submission_date
 GROUP BY
-  submission_date,
-  sample_id,
-  client_id
+  1,
+  2,
+  3
```

Comment:
```suggestion
  submission_date,
  sample_id,
  client_id
```

Our preferred style is to write out column names explicitly in group by and sort by. I am failing to find this practice documented, though.